frontend/src/__tests__/integration/pages/modelServing/ServingRuntimeList.spec.ts

```diff
@@ -31,3 +31,27 @@ test('Deploy model', async ({ page }) => {
     .fill('test-secret-key');
   await expect(await page.getByRole('button', { name: 'Deploy', exact: true })).toBeEnabled();
 });
+
+test('Legacy Serving Runtime', async ({ page }) => {
```

Comment:
You don't have a test for one that is NOT OVMs to make sure we are not blind converting...

Reply:
Yeah, I'm just gonna update it with one for caikit

Reply:
[UPDATE]: Since i need several runtimes to cover all the functionality for model serving I'll leverage adding that when I add coverage for all the Model Serving feature.